### DISCUSSION TITLE
Will local hosts file interfere with uBlock Origin?

### DISCUSSION THREAD

**Discussion by partingscientist** [uBlockOrigin/uBlock-issues#2455] [Q&A] [answered]
How would such scenario (if there is any) look like? I found [this](https://www.reddit.com/r/uBlockOrigin/comments/khkysa/is_using_a_hosts_file_block_list_in_windows_on/) while looking this up, but I'm still not sure, specifically on how uBO's exception would interact with a hosts file entry. 

Thank you in advance.

**Comment by Yuki2718** [comment]
If a domain is exempted on uBO for some problem but blocked on your hosts, you'll see the problem. uBO's exception only works on uBO.

**Comment by uBlock-user** [comment, marked as answer]
uBO only has access to extension frameworks, beyond that any intervention from third-party actors will and can cause issues related to blockage.
